deploy/eck-elastic-agent/examples/system-integration.yaml

```diff
@@ -0,0 +1,131 @@
+version: 8.2.3
```

Comment:
I would add a comment to explain that this should only be used in "standalone" mode. (or maybe add a `README.md` similar to this [one](https://github.com/elastic/cloud-on-k8s/blob/main/config/recipes/elastic-agent/README.asciidoc))

Reply:
I've added comments to both examples noting when they should be used.